website/source/docs/drivers/docker.html.md

```diff
@@ -125,8 +125,10 @@ production Nomad will always read `docker.endpoint`.
 
 The `docker` driver will set the following client attributes:
 
-* `driver.docker` - This will be set to "1", indicating the
+* `driver.docker` - This will be set to "true", indicating the
```

Comment:
This is actually a mistake. I will go through and update the code for the drivers to all be 1. It is only docker and Rkt that do this.